docs/Customization.md

```diff
@@ -0,0 +1,22 @@
+# Customization
+
+## GraphQL Server
+
+The LikeDAO backend uses a dotenv file that contains server configurations in the form of environemtn variables. This file is for **development use** only where a `.env` file will be created from the `.env.example` file upon running `make setup` in the server's root folder.
+
+Note that the file contains configurations specific to the Cosmos-based blockchains so it should be configured to match the selected chain for the server to function correctly.
+
+## React App
+
+The LikeDAO frontend has two files that contain configurations specific to a Cosmos-based blockchain.
+
+After running `make setup`, two extra files with the suffix `debug.js` will appear in the `./config` folder.
+
+`config.debug.js` contains common configurations that are used among components and providers in the app.
+`keplr.debug.js` contains neccessary information for a chain that will be used by the Keplr wallet extension via their [Suggest Chain API](https://docs.keplr.app/api/suggest-chain.html)
+
+These files are specificly for **development use** and will be copied to the `./public` folder when `make dev` is executed so make sure the files are well-prepared before any development process is started.
+
+## BDJuno
```

Comment:
when adopting to other chains, is switching to another bdjuno fork needed?

Reply:
oh probably yes